.NET/Microsoft.Recognizers.Text.Number/Extractors/BaseNumberExtractor.cs

```diff
@@ -138,9 +141,15 @@ public class LongFormatType
         // 1 234 567,89
         public static LongFormatType DoubleNumBlankComma = new LongFormatType(' ', ',');
 
+        // 1 234 567,89
+        public static LongFormatType DoubleNumNoBreakSpaceComma = new LongFormatType(' ', ',');
+
         // 1 234 567.89
         public static LongFormatType DoubleNumBlankDot = new LongFormatType(' ', '.');
 
+        // 1 234 567.89
+        public static LongFormatType DoubleNumNoBreakSpaceDot = new LongFormatType(' ', '.');

```

Comment:
It would be nice to define a constant somewhere to represent this special space char. And add some code comment for people to realize what it is. Even better if we can write the unicode code and convert, so file encoding issues don't break it.